ipams/remote/remote_test.go

```diff
@@ -61,6 +61,35 @@ func setupPlugin(t *testing.T, name string, mux *http.ServeMux) func() {
 	}
 }
 
+func TestGetCapabilities(t *testing.T) {
```

Comment:
Please also add test for IPAM drivers that doesnt implement `GetCapabilities` and how we treat them ?